core/chains/sui.go

```diff
@@ -0,0 +1,179 @@
+package chains
+
+import (
+	"bytes"
+	"encoding/json"
+	"fmt"
+	"io"
+	"log"
+	"math/big"
+	"net/http"
+	"sort"
+	"strconv"
+
+	utils "github.com/xenowits/nakamoto-coefficient-calculator/core/utils"
+)
+
+type SuiRespons struct {
+	Jsonrpc string `json:"jsonrpc"`
+	Result  struct {
+		Epoch                                 string `json:"epoch"`
+		ProtocolVersion                       string `json:"protocolVersion"`
+		SystemStateVersion                    string `json:"systemStateVersion"`
+		StorageFundTotalObjectStorageRebates  string `json:"storageFundTotalObjectStorageRebates"`
+		StorageFundNonRefundableBalance       string `json:"storageFundNonRefundableBalance"`
```

Comment:
Remove all fields that are not required from this struct since the only relevant field is `VotingPower`